### DISCUSSION TITLE
Support for Flutter Desktop Windows

### DISCUSSION THREAD

**Discussion by longtimedeveloper** [firebase/flutterfire#5854] [Feature request] [answered]
Hi,

I can't find docs for Flutter Desktop Windows applicaitons.  Is this in the roadmap?

Thank you,

Karl

**Comment by longtimedeveloper** [comment]
Found this package:  https://pub.dev/packages/firedart

**Comment by barbalex** [comment, marked as answer]
Duplicate of https://github.com/FirebaseExtended/flutterfire/discussions/5557
